IPs/RSKIP_tbd.md

```diff
@@ -0,0 +1,282 @@
+---
+rskip: pull_request_number_here
+title: Powpeg Spendability Validation Protocol
+created: 31-AUG-23
+author: JD
+purpose: Sec
+layer: Core
+complexity: 2
+status: Draft
+description: New Mechanism for current Powpeg to ensure the newly elected Powpeg will safely hold user funds
+---
+
+|RSKIP          |pull_request_number_here           |
+| :------------ |:-------------|
+|**Title**      |Powpeg Spendability Validation Protocol |
+|**Created**    |31-AUG-23 |
+|**Author**     |JD |
+|**Purpose**    |Sec|
+|**Layer**      |Core|
+|**Complexity** |2|
+|**Status**     |Draft|
+
+## Abstract
+
+The Powpeg composition changes from time to time for different reasons, mainly members joining or leaving and/or POWHSM upgrades that require onboarding.
+The Powpeg Bitcoin address also changes when the redeemscript is upgraded, for instance when the ERP redeemscript was added.
+Everytime this happens, the Bridge undergoes a process that is irreversible, after a certain amount of blocks, the new Powpeg composition will receive all the funds and become responsible for its safety. If the newly elected Powpeg were to be unable to securely store the funds, the users could have their funds at risk.
+
+This RSKIP proposes a new process that will happen before the new Powpeg takes over, and could imply reverting the Powpeg election if the new Powpeg is considered not safe.
+
+## Motivation
+
+The Powpeg election is a critical process that should have as many reassurances as possible.
+As mentioned in the previous section, nowadays once the election takes place, the process is irreversible.
+This RSKIP proposes an automated and decentralized process to add one more layer of security to this process to avoid any human mistake putting funds at risk.
+
+This process protects funds from:
+- An invalid redeemscript that would render the funds burned.
+- A composition of pegnatories that are not responsive.
+
+This process doesn't include:
+- Verifying proposed new members use a valid HSM version.
+
+## Specification
+
+This RSKIP proposes changes in the Powpeg change process. The subsections will expand on the changes for each step.
+
+The election process in itself doesn't change. The authorized members will still propose a new composition, once the voting finishes the new composition will be committed and the Powpeg change will start.
+
+The current process consists in the following phases:
+1. Election start: m/n authorized members propose to start a new election.
+2. Composition proposal: m/n vote for the new composition.
+3. New composition commitment: m/n vote to commit new composition. New elections are disabled.
+4. Pre activation phase: immediately after commitment and during a number of blocks. No real change for end users.
```

Comment:
should we mention what is the pre activation phase for? or we assume that anyone who's reading this knows it?